ruby_oop/tic_tac_toe/game.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require './player'
+require './board'
+
+# class to set up the game
+class Game
+  attr_accessor :board, :player1, :player2
+
+  def initialize
+    @player1 = create_player
+    @player2 = create_player(ordinal_prompt: 'second')
+    @board = Board.new
+    board.display_board
+  end
+
+  def swap_players
+    turn_number = 1
+    until game_over?
+      if turn_number.odd?
+        puts "#{player1.name.capitalize} is selecting a spot..."
+        @board.board_update(player1.marker)
+        turn_number += 1
+      else
+        puts "#{player2.name.capitalize} is selecting a spot..."
+        @board.board_update(player2.marker)
+        turn_number += 1
+      end
+    end
+  end
```

Comment:
I took a stab at refactoring `swap_players` into two methods, `play` and `swap_players`, in order to:

1. create a public method called `play` which I think reads better as `@game.play` vs. `@game.swap_players`. As a user of this class, I'm wondering why I need to call `swap_players` in order to "play" Tic Tac Toe.  Ya know?
2. DRYs up assigning `turn_number += 1`.
3. Makes `swap_players` private as that seems more like an internal concern of the class (see 1 above)

```suggestion
 
   def play
     turn_number = 1
     until game_over?
       swap_players(turn_number)
       turn_number += 1
     end
   end
 
   private
  
   def swap_players(turn_number)
     if turn_number.odd?
       puts "#{player1.name.capitalize} is selecting a spot..."
       @board.board_update(player1.marker)
     else
       puts "#{player2.name.capitalize} is selecting a spot..."
       @board.board_update(player2.marker)
     end
   end
```

What do you think? Happy to pair on this if I wasn't clear.